src/Form/NewTalkType.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Form;
+
+use App\Entity\User;
+use App\Form\EventSubscribers\CreateTalkSubscriber;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class NewTalkType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('title', TextType::class)
+            ->add('intro', TextareaType::class)
+            ->add('conference', ConferenceDatalistType::class, [
+                'mapped' => false,
+            ])
+            ->add('users', EntityType::class, [
+                'class' => User::class,
+                'multiple' => true,
+                'mapped' => false,
+            ])
+            ->addEventSubscriber(new CreateTalkSubscriber())
```

Comment:
Should use dependency injection (but see the other comment before, since everything will be moved to this class)